src/odemis/gui/cont/stream_bar.py

```diff
@@ -1198,6 +1200,51 @@ class SecomStreamsController(StreamBarController):
     Controls the streams for the SECOM and DELPHI live view
     """
 
+    def __init__(self, *args, **kwargs):
+        dir = os.path.join(get_home_folder(), u".config/odemis")
+        file_path = os.path.abspath(os.path.join(dir, f"stream_settings.json"))
+        max_entries = 10  # Maximum number most recent settings to display
+        # Initialize a file to save acquired streams
+        self.stream_settings = StreamSettings(file_path, max_entries)
+        super().__init__(*args, **kwargs)
+
+    def update_stream_settings(self):
+        """
+        Update the recently used stream setting when the stream settings are updated during an acquisition
+        """
+        streams = [s.stream for s in self.stream_controllers]
+        self.stream_settings.get_streams_settings(streams)
+
+        if self.stream_settings.config_data:
+            acq_names = sorted([e["name"] for e in self.stream_settings.config_data])
```

Comment:
It's not required, but I feel it'd a nicer interface if you provided the list of entries (sorted in MRU) on the `StreamSettingsConfig` class, as a read-only attribute. It's just a matter of adding something like:
```
@property
def entries(self):
   return [e["name"] for e in self.stream_settings.config_data]
```